cluster/charts/rook-ceph/templates/psp.yaml

```diff
@@ -58,8 +58,6 @@ spec:
   hostIPC: true
   # Ceph OSDs need to share the same PID namespace
   hostPID: true
-  # hostNetwork can be set to 'false' if host networking isn't used
```

Comment:
this not related to cephcsi. we should not remove it.

Reply:
okay